lib/api/controller/auth.js

```diff
@@ -286,6 +286,8 @@ class AuthController extends NativeController {
    * @returns {Promise<object>}
    */
   getMyRights (request) {
+    request.addDeprecation('2.4.0', 'http://kuzzle:7512/users/_me/_rights as been replaced by http://kuzzle:7512/_me/_rights');
```

Comment:
This deprecation warning will be printed every time this action is called, even with WebSocket and with the new HTTP route.

You should ensure to add the deprecation only when the deprecated route has been used.

Maybe it could be interesting to add a new property in the `httpRoutes.js` definition file and handle those deprecation in the funnel level instead of adding `if` in controller methods.

Maybe something like
```
{ verb: 'get', url: 'http://kuzzle:7512/users/_me/_rights', controller: 'auth', action: 'getMyRights', deprecated: { since: '2.4.0', message: 'http://kuzzle:7512/_me/_rights' } }
```

Reply:
That's sounds good 👍